cg/cli/deliver/base.py

```diff
@@ -133,60 +51,56 @@ def rsync(context: CGConfig, ticket: str, dry_run: bool):
     )
 
 
-@deliver.command(name="concatenate")
-@DRY_RUN
-@TICKET_ID_ARG
-@click.pass_context
-def concatenate(context: click.Context, ticket: str, dry_run: bool):
-    """The fastq files in the folder generated using "cg deliver analysis"
-    will be concatenated into one forward and one reverse fastq file
+@deliver.command(name="case")
+@DELIVERY_TYPE
+@click.option(
+    "-c",
+    "--case-id",
+    required=True,
+    help="Deliver the files for a specific case",
+)
+@click.pass_obj
+def deliver_analysis(
+    context: CGConfig,
+    case_id: str,
+    delivery_type: list[str],
```

Comment:
delivery_type given here is "mid-dna", "balsamic" etc. It doesn't make much sense but it's what prod uses, we'll have to ask them if they want to keep it

Reply:
I would say change it to workflow

Reply:
Removed the option. Delivery type and workflow are set on case. I will fetch it from there.